addons/default/visiosoft/defaultadmin-theme/resources/views/table/partials/footer.twig

```diff
@@ -0,0 +1,62 @@
+<tfoot>
+{% if table.actions|length or table.data.pagination.links|length %}
+	<tr>
+		<th colspan="100%" style="padding: 10px;">
+
+			<div class="pull-left actions table__actions">
+				{{ buttons(table.actions)|raw }}
+			</div>
+
+			{% if table.data.pagination.links|length %}
+				<div class="pull-right">
+
+					<select onchange="window.location=this.value;"
+					        class="custom-select table-limit">
+						<option {{ table.options.limit == 5 ? 'selected' }}
+							value="{{ url_current() }}?{{ http_build_query({(table.options.prefix ~ 'limit'): 5} + request_query()) }}">
```

Comment:
can you make it as function?

url_current() }}?{{ http_build_query({(table.options.prefix ~ 'limit'): 5} + request_query())